qx/handle_difficulty.go

```diff
@@ -51,12 +52,12 @@ func CompactToGPS(diff string,edgeBits ,blockTime ,height int) {
 	fmt.Printf("%f\n", needGPS)
 }
 
-func GPSToDiff(gps string,edgeBits ,blockTime,height int) {
+func GPSToDiff(gps string,edgeBits ,blockTime,height int,p *params.Params) {
 	f64,err := strconv.ParseFloat(gps,64)
 	if err != nil{
 		ErrExit(err)
 	}
-	scale := pow.GraphWeight(uint32(edgeBits),int64(height),pow.CUCKAROO)
+	scale := pow.GraphWeight(uint32(edgeBits),int64(height),p.PowConfig.AdjustmentStartMainHeight,pow.CUCKAROO)
```

Comment:
same reason above, qx should not aware of `p.PowConfig.AdjustmentStartMainHeight`